src/mixins/allowable-actions-mixin.ts

```diff
@@ -165,7 +166,7 @@ export default class AllowableActionsMixin extends Vue {
       }
 
       case AllowableActions.STAFF_COMMENT: {
-        return (isBusiness && this.isRoleStaff)
+        return (isBusiness && this.IsAuthorized(AuthorizedActions.STAFF_COMMENTS))
```

Comment:
We need separate authorized actions for staff comments on filings and staff comments on the business.

Ref:
![image](https://github.com/user-attachments/assets/570eabab-dfea-4c91-8651-9bb30613e157)

Reply:
Oops, we no longer handle detail comments on filings in this UI. Resolving.